pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```diff
@@ -3,28 +3,15 @@
 let
   # These names are how they are designated in https://xanmod.org.
   ltsVariant = {
-    version = "5.15.70";
-    hash = "sha256-gMtGoj/HzMqd6Y3PSc6QTsu/PI7vfb+1pg4mt878cxs=";
+    version = "5.15.75";
+    hash = "sha256-tgm5nmguEfRFq3OhmZgRgFLIW7E798Rv1basxnfdqLI=";
     variant = "lts";
   };
 
-  currentVariant = {
-    version = "5.19.13";
-    hash = "sha256-BzQH4c24CtE3R5HNe2sOc3McVkRmf/RKOOjuf1W4YfE=";
-    variant = "current";
-  };
-
-  nextVariant = {
-    version = "6.0.0";
-    hash = "sha256-E7T8eHwMKYShv4KWdCbHQmpn+54edJoKdimZY3GFbPU=";
-    variant = "next";
-  };
-
```

Comment:
We might want to add an entry to the release notes for this.

Note that we actually changed names twice since 22.05: https://github.com/NixOS/nixpkgs/blob/nixos-22.05/pkgs/os-specific/linux/kernel/xanmod-kernels.nix

Reply:
I don't know about that, these renames don't affect how the end-user uses the package and we're purely for the intention of knowing which one to check the version on the website.

And `linux_xanmod` has always been LTS during 22.05 and `linux_xanmod_latest` has always been the latest available.